src/pages/home/components/Sidebar/Sidebar.module.scss

```
@@ -0,0 +1,39 @@
+* {
+  box-sizing: border-box;
+}
+
+.sidebar {
+  width: 100%;
+
+  font-size: 1.6rem;
+
+  border: 1px solid #cbc4cf;
+
+  background-color: #000;
+
+  background-color: #f5eff4;
```

```suggestion
  background-color: #000;

  background-color: #f5eff4;
```

Tem dois background-color.